sprocs/assessment_instances_insert.sql

```diff
@@ -26,19 +29,50 @@ BEGIN
 
     -- ######################################################################
     -- determine the "number" of the new assessment instance
+    IF group_work THEN
+        SELECT
+            gu.group_id
+        INTO
+            tmp_group_id
+        FROM
+            group_users as gu
+            JOIN groups AS g ON (g.id = gu.group_id)
+            JOIN group_configs AS gc ON (gc.id = g.group_config_id)
+        WHERE 
+            gu.user_id = assessment_instances_insert.user_id
+            AND gc.assessment_id = assessment_instances_insert.assessment_id
+            AND gc.deleted_at IS NULL
+            AND g.deleted_at IS NULL;
+            IF NOT FOUND THEN
```

Comment:
Indented too far -- should be at the same left as the SELECT. Also please leave a blank line before the `IF NOT FOUND` statement.